include/hip/nvcc_detail/hip_runtime_api.h

```diff
@@ -351,25 +358,24 @@ inline static hipError_t hipMemcpy(void* dst, const void* src, size_t sizeBytes,
 }
 
 
-inline static hipError_t hipMemcpyAsync(void* dst, const void* src, size_t sizeBytes, hipMemcpyKind copyKind, hipStream_t stream=0) {
-  return hipCUDAErrorTohipError(cudaMemcpyAsync(dst, src, sizeBytes, hipMemcpyKindToCudaMemcpyKind(copyKind), stream));
+inline static hipError_t hipMemcpyAsync(void* dst, const void* src, size_t sizeBytes, hipMemcpyKind copyKind, hipStream_t stream __dparm(0)) {
+  return hipCUDAErrorTohipError(cudaMemcpyAsync(dst, src, sizeBytes, hipMemcpyKindToCudaMemcpyKind(copyKind), 0));
```

Comment:
Should still remain ```stream`` and not ```0```. Otherwise we cannot queue a cudaMemcpyAsync to any other stream other than NULL stream.

Reply:
@wsttiger - can you make the change Maneesh suggested and also rebase on master to pick up a test env change from kent?